keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -472,13 +472,16 @@ Because Windows privileged containers will work much differently than Linux priv
 ![Privileged Container Diagram](Privileged.png)
 
 #### Networking
+
 - The container will be in the host’s network namespace (default network compartment) so it will have access to all the host’s network interfaces and have the host's IP as well.
 
 #### Resource Limits
+
 - Resource limits (disk, memory, cpu count) will be applied to the job and will be job wide. For example, with a limit of 10 MB is set for the job, if every process in the jobs memory allocations added up exceeds 10 MB this limit would be reached. This is the same behavior as other Windows container types. These limits would be specified the same way they are currently for whatever orchestrator/runtime is being used.
-- Disk resource tracking may work slightly differently for privileged Windows containers due to how these containers are bootstrapped. The extent of these differences are still being investigated but will be fully documented when understood. Resource usage will be trackable the differences would be in how resource usage is calculated.
+- Disk resource tracking may work slightly differently for `hostProcess` containers due to how these containers are bootstrapped. Resource usage will be trackable and the differences would be in how resource usage is calculated.
```

Comment:
We may not be able to attribute the disk usage directly to the container?  Maybe the new approach will help?

Reply:
Could someone clarify disk resource tracking here? What do we want tracked?

Reply:
Kubelet exposes some metrics for mounted volumes around capacity so that operators can notice when storage is fulling up for their workloads or potentially resize the volumes.

The best info I can find on this is at https://github.com/kubernetes/community/pull/855/files?short_path=abd8530#diff-abd85306506ef043bca017e5d3b489718ca00d77a2d3f7d3b0878558fe2dc763

We'll want to document behaviors here and encourage operators to track the total free space on the node.